vendor/openvino/openvino.BUILD

```diff
@@ -0,0 +1,151 @@
+        "@MKLDNN_VERSION_MINOR@": "1",
+        "@MKLDNN_VERSION_PATCH@": "1",
+        "@MKLDNN_VERSION_HASH@": "afd",
+    },
+    src = "inference-engine/thirdparty/mkl-dnn/include/mkldnn_version.h.in",
+    out = "inference-engine/thirdparty/mkl-dnn/include/mkldnn_version.h",
+)
+
+cc_library(
+    name = "mkldnn_plugin",
+    alwayslink = True,
+    srcs = glob([
+        "inference-engine/thirdparty/mkl-dnn/src/*pp",
+        "inference-engine/thirdparty/mkl-dnn/src/**/*pp",
+        "inference-engine/src/mkldnn_plugin/*pp",
+        "inference-engine/src/mkldnn_plugin/**/*pp",
+    ]),
+    hdrs = glob([
+        "inference-engine/thirdparty/mkl-dnn/include/*",
+        "inference-engine/thirdparty/mkl-dnn/src/common/*.hpp",
+        "inference-engine/thirdparty/mkl-dnn/src/cpu/*.hpp",
+        "inference-engine/thirdparty/mkl-dnn/src/cpu/**/*.h*",
+        "inference-engine/thirdparty/mkl-dnn/src/*.hpp",
+        "inference-engine/src/mkldnn_plugin/**/*hpp",
+    ]) + [":mkldnn_version"],
+    includes = [
+        "inference-engine/thirdparty/mkl-dnn/include",
+        "inference-engine/thirdparty/mkl-dnn/src/cpu",
+        "inference-engine/thirdparty/mkl-dnn/src/common",
+        "inference-engine/thirdparty/mkl-dnn/src",
+        "inference-engine/src/mkldnn_plugin/",
+        "inference-engine/src/mkldnn_plugin/mkldnn",
+    ],
+    deps = [ "inference_engine" ],
+)
+
+cc_library(
+    name = "inference_engine",
+    alwayslink = True,
+    srcs = glob(
+        [
+            "inference-engine/src/extension/*pp",
+            "inference-engine/src/extension/**/*pp",
+            "inference-engine/src/extension/common/*",
+            "inference-engine/src/inference_engine/*.cpp",
+            "inference-engine/src/inference_engine/builders/*pp",
+            "inference-engine/src/inference_engine/dumper/*.cpp",
+            "inference-engine/src/inference_engine/ngraph_ops/*pp",
+            "inference-engine/src/inference_engine/cpp_interfaces/*.*pp",
+            "inference-engine/src/inference_engine/shape_infer/*.cpp",
+            "inference-engine/src/inference_engine/shape_infer/**/*.cpp",
+            "inference-engine/src/preprocessing/*.cpp",
+            "inference-engine/samples/*.cpp",
+            "inference-engine/samples/common/**/*.cpp",
+        ],
+    ),
+    hdrs = glob(
+        [
+            "inference-engine/src/extension/*.hpp",
+            "inference-engine/src/extension/common/*",
+            "inference-engine/include/**/*.h*",
+            "inference-engine/samples/*.hpp",
+            "inference-engine/samples/common/**/*.h*",
+            "inference-engine/src/inference_engine/*.h",
+            "inference-engine/src/inference_engine/cpp_interfaces/*.h*",
+            "inference-engine/src/inference_engine/shape_infer/*.h*",
+            "inference-engine/src/inference_engine/shape_infer/**/.h*",
+            "inference-engine/src/preprocessing/*.h*",
+        ],
+    ),
+    includes = [
```

Comment:
Can we make this local includes? Possibly with `copts = ["-I"]`, because I'm worried about how this will interact with downstream targets.

Reply:
understood, difficult right now and there should be very few downstream targets - some of them are required in some circumstances which may extend to building engines